server/src/main/java/io/druid/segment/indexing/DataSchema.java

```diff
@@ -58,6 +59,7 @@ public DataSchema(
       @JsonProperty("parser") Map<String, Object> parser,
       @JsonProperty("metricsSpec") AggregatorFactory[] aggregators,
       @JsonProperty("granularitySpec") GranularitySpec granularitySpec,
+      @JsonProperty("rollup") Boolean rollup,
```

Comment:
Okay, I think that is reasonable. Can we please put it in the granularity spec then? The configuration to define the rollup right now (queryGranularity) is there